### PR TITLE
Added search_folders - returns folder names | Added send_telegram functionality to notify user when some task is done

### PR DESCRIPTION
**Pull request by Reno-Codes**
### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have thoroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] I have documented my changes clearly and comprehensively.
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by Reno-Codes** [comment]
just added search_folders command, AutoGPT wasn't able to retrieve the list of folders if asked to do so

**Comment by Amraki** [comment]
The command "search_folders" works for me so far.

Unfortunately, this PR is also deleting .env.template so you'll want to correct that.

**Review by Reno-Codes — COMMENTED** [review]
Forgot to include .env.template

File: .env.template [hunk 1]
```diff
@@ -170,4 +170,4 @@ ELEVENLABS_VOICE_2_ID=your-voice-id-2
 TW_CONSUMER_KEY=
 TW_CONSUMER_SECRET=
 TW_ACCESS_TOKEN=
-TW_ACCESS_TOKEN_SECRET=
+TW_ACCESS_TOKEN_SECRET=
```
Comment (Pwuts) [inline .env.template]: Please entirely revert change to this file. Should not be included in the PR.

**Review by Pwuts — CHANGES_REQUESTED** [review]
Nice addition :)

File: .env.template (hunk 1 above)
Comment (Reno-Codes) [reply .env.template]: Thank you, but I just added send_telegram functionality which required making changes to .env.template, could you review it and let me know what you think? :)

**Review by Pwuts — CHANGES_REQUESTED** [review]
Please do not combine unrelated feature additions in 1 PR. Something like Telegram integration is more suited for a plugin (#757), and won't be merged into the core project.

**Comment by Reno-Codes** [comment]
> Please do not combine unrelated feature additions in 1 PR. Something like Telegram integration is more suited for a plugin (#757), and won't be merged into the core project.

I know, but then Twitter is more suitable for a plugin as well.
Sorry, it's my first time trying to contribute, haven't used GitHub for this purpose before
